src/index.css

```diff
@@ -0,0 +1,342 @@
+body {
```

Comment:
Seems that you are using styles from here. How come you decided to use this approach?

Reply:
Before I answer, I have to say that usually CSS in a layout takes a lot of my time.
For this project I decided to make every CSS addition in the last 2 days to avoid wasting precious time and focus on the functionality. 
I decided to move all my CSS code to one place because I was in hurry to finish everything and because I though it was not so big of an app to actually modularize it.